lib/linters/README.md

```diff
@@ -835,6 +836,24 @@ Option     | Description
 }
 ```
 
+## universalSelector
+Disallow the usage of the universal selector.
+The universal selector shouldn't be used since it'll cancel out any inheritance.
+
+### invalid
+```less
+* {
+    color: red;
+}
+```
```

Comment:
consider add here another example with sub universal selector `.foo * {}` or change the * {} to a sub selector. It is not clear that also sub universal selectors are not allowed.